calyx-opt/src/passes/compile_repeat.rs

```diff
@@ -103,10 +103,10 @@ impl Visitor for CompileRepeat {
                 None,
                 Box::new(while_body),
             );
-            let while_seq = ir::Control::seq(vec![
-                ir::Control::enable(init_group),
-                while_loop,
-            ]);
+            let while_seq = ir::Control::Seq(ir::Seq {
```

Comment:
Is there an `ir::Control::seq` method to construct this nest of `Seq { Seq { ... } } `?

Reply:
There is, but it gives the `seq` `ir::Attributes::default()`.